app/controllers/application_controller.rb

```diff
@@ -15,12 +15,18 @@ def return_current_user_projects
     end
   end
 
+  def generate_thumbnail(path, imagefile, commit_id)
+    image = Magick::Image.read("#{path}/satellite/#{imagefile.original_filename}").first
+    image.scale(50,50).write "#{path}/thumbnails/#{commit_id}"
+  end
+
```

Comment:
yup, you're right - you can move them to the projects controller. i'd love it if you could also watch out for similar nits and move them to the better location at your discretion. just remember to leave a note on those when you send PRs, so we don't overlook - for most of the part though, travis should tell us :)